### PR TITLE
Adds position and velocity interfaces to FrankaHWSim

### PR DESCRIPTION
**Pull request by rickstaa**
This commit adds the 'hardware_interface/PositionJointInterface' and 'hardware_interface/VelocityJointInterface' interfaces to the
FrankaHWSim module. This was done to make the simulation closer match the real robot. 

#### Remarks

- I didn't add the position and velocity controllers to the [sim_controllers.yaml](https://github.com/rickstaa/franka_ros/blob/adds_position_velocity_interfaces_gazebo/franka_gazebo/config/sim_controllers.yaml) file, as these example controllers expect the robot to be at the home position. Users can however, with this commit load their own position and velocity controllers.
- The [FrankaHWSim documentation](https://frankaemika.github.io/docs/franka_ros.html#frankahwsim) of course needs to be updated if this pull request is merged.
- Best to be merged with https://github.com/frankaemika/franka_ros/pull/157 since this allows users to start the simulation with different hardware_interfaces.

### CODE REVIEW

**Review by tahsinkose — APPROVED** [review]
Giving a rogue review. Please do not push format changes unless they are required @rickstaa. They make finding the actual changes quite difficult and also create unnecessarily large diffs.

The rest LGTM. As I'm not a maintainer, my approval won't make a difference, but here it is as a goodwill gesture 😄

**Comment by rickstaa** [comment]
@tahsinkose Ah, thanks for pointing that out. I will rebase this commit, I forgot to disable the auto-format pre-commit pipeline for this one. I will perform a rebase to fix it.

**Comment by rickstaa** [comment]
Please read https://github.com/frankaemika/franka_ros/issues/161#issuecomment-921837993 before merging #156. There I explain why I think this pull request is not the best solution.

**Comment by rickstaa** [comment]
Closed since it is replaced by #181.

**Pull request closed without merging**
